config/locales/pt-BR.yml

```diff
@@ -0,0 +1,556 @@
+      cebola_sp: cebola sp
+      agrião: agrião
+      almeirão: almeirão
+      beterraba_com_folhas: beterraba com folhas
+      brócolis: brócolis
+      catalonha: catalonha
+      cenoura_com_folhas: cenoura com folhas
+      couve_de_bruxelas: couve de bruxelas
+      couve-flor: couve-flor
+      erva-doce: erva-doce
+      espinafre: espinafre
+      folha_de_uva: folha de uva
+      gobo: gobo
+      louro: louro
+      manjericão: manjericão
+      mostarda: mostarda
+      nabo: nabo
+      rabanete: rabanete
+      rúcula: rúcula
+      abacate_breda/margarida: abacate breda/margarida
+      abacaxi_pérola: abacaxi pérola
+      acerola: acerola
+      amêndoa: amêndoa
+      amora: amora
+      avelã: avelã
+      banana_prata: banana prata
+      castanha_nacional: castanha nacional
+      coco_verde: coco verde
+      graviola: graviola
+      jaca: jaca
+      laranja_pera: laranja pera
+      lima_da_pérsia: lima da pérsia
+      maçã_nacional_fuji: maçã nacional fuji
+      maçã_estrangeira_red_delicious: maçã estrangeira red delicious
+      mamão_havaí: mamão havaí
+      manga_haden: manga haden
+      manga_palmer: manga palmer
+      manga_tommy: manga tommy
+      maracujá_doce: maracujá doce
+      melão_amarelo: melão amarelo
+      nectarina_nacional: nectarina nacional
+      nozes: nozes
+      sapoti: sapoti
+      tangerina_murcot: tangerina murcot
+      uva_thompson: uva thompson
+      abobrinha_italiana: abobrinha italiana
+      berinjela_comum: berinjela comum
+      berinjela_japonesa: berinjela japonesa
+      maxixe: maxixe
+      pepino_japonês: pepino japonês
+      pepino_caipira: pepino caipira
+      taquenoco: taquenoco
+      tomate_caqui: tomate caqui
+      cebola_nacional: cebola nacional
+      cebola_sp: cebola sp
+      almeirão: almeirão
+      aspargos: aspargos
+      beterraba_com_folhas: beterraba com folhas
+      brócolis: brócolis
+      cenoura_com_folhas: cenoura com folhas
+      couve_de_bruxelas: couve de bruxelas
+      erva-doce: erva-doce
+      espinafre: espinafre
+      folha_de_uva: folha de uva
+      gobo: gobo
+      hortelã: hortelã
+      manjericão: manjericão
+      mostarda: mostarda
+      nabo: nabo
+      palmito: palmito
+      rabanete: rabanete
+      abacaxi_havaí: abacaxi havaí
+      abacaxi_pérola: abacaxi pérola
+      acerola: acerola
+      ameixa_nacional: ameixa nacional
+      amêndoa: amêndoa
+      amora: amora
+      castanha_nacional: castanha nacional
+      cereja_estrangeira: cereja estrangeira
+      castanha_estrangeira: castanha estrangeira
+      coco_verde: coco verde
+      damasco_estrangeiro: damasco estrangeiro
+      figo: figo
+      framboesa: framboesa
+      graviola: graviola
+      kiwi_estrangeiro: kiwi estrangeiro
+      laranja_pera: laranja pera
+      lichia: lichia
+      limão_taiti: limão taiti
+      maçã_nacional_fuji: maçã nacional fuji
+      maçã_estrangeira_red_delicious: maçã estrangeira red delicious
```

Comment:
Keys should not contain special characters. So let's restrict to `a-z A-Z 0-9 _ -`. Thus:
- `maçã_estrangeira_red_delicious` becomes `maca_estrangeira_red_delicious`
- `abacate_breda/margarida` becomes `abacate_breda_margarida`
- `abóbora_d'água` becomes `abobora_d_agua`
- etc